src/Hooks/MainHookHandler.php

```diff
@@ -78,13 +78,13 @@ public function onSkinBuildSidebar( $skin, &$bar ) {
 							}
 						} elseif ( array_key_exists( 'iconfile', $tournament ) ) {
 							$iconfileTitle = Title::newFromText(
-									$iconTemplatePrefix . '/mainpageTST',
+									$iconTemplatePrefix . '/custom',
 									NS_TEMPLATE
 							);
 							if ( $iconfileTitle !== null && $iconfileTitle->exists() && $skin->getTitle() !== null ) {
```

Comment:
As long as the `$iconfileTitle->exists()` code is in there it will not see the commons version existing.